src/main/java/com/thealgorithms/datastructures/lists/CircleDoublyLinkedList.java

```diff
@@ -0,0 +1,356 @@
+package com.thealgorithms.datastructures.lists;
+
+public class CircleDoublyLinkedList<E> {
```

Comment:
Provide a link to wikipedia or any other source that explains the datastructure in a javadoc comment.